rootfs/api/models/config.py

```diff
@@ -46,7 +46,8 @@ def _migrate_legacy_healthcheck(self):
         success_threshold = int(self.values.get('HEALTHCHECK_SUCCESS_THRESHOLD', 1))
         failure_threshold = int(self.values.get('HEALTHCHECK_FAILURE_THRESHOLD', 3))
 
-        self.healthcheck['livenessProbe'] = {
+        self.healthcheck[''] = {}
```

Comment:
Empty is a little weird, but we are only doing that to have a "global" proc type (so to speak)? Can we use `__all` or something similar that our proc name verifier would otherwise disallow when a user passes it in?